src/hooks/use-authentication.ts

```diff
@@ -57,8 +57,11 @@ const useAuthentication = (options: UseAuthenticationOptions = {}) => {
 	})
 
 	// Deriving booleans about auth state
+	const isAuthEnabled = AUTH_ENABLED
 	const isLoading = status === 'loading'
 	const isAuthenticated = status === 'authenticated'
+	const showAuthenticatedUI = isAuthenticated
```

Comment:
This one might feel a bit redundant since it just returns `isAuthenticated`, but do see how it would read better to have these two options. Just thinking outloud.